src/api/sso/src/authentication.js

```diff
@@ -116,27 +115,29 @@ const strategy = new SamlStrategy(
 
     async function lookupTelescopeUser(id) {
       // We have the user's id (i.e. their hashed email address). Now
-      // make a service-to-service request to the Users service in order to
+      // make a request to the Supabase service in order to
       // get this user's full profile information using the user's id.  They
       // may or may not have a Telescope profile (yet).
       try {
-        const res = await fetch(`${USERS_URL}/${id}`, {
-          headers: {
-            Authorization: `bearer ${createServiceToken()}`,
-          },
-        });
-        if (!res.ok) {
-          if (res.status === 404) {
-            // No Telescope user profile found, so create a regular Seneca user
-            logger.debug({ senecaProfile }, `No Telescope account for ${id} with Users service`);
-            done(null, new User(senecaProfile));
-            return;
-          }
-          // We can't get a response from the Users service, so we don't know what we have.
-          throw new Error(`unable to get user info from Users service: ${res.status}`);
+        const { data: profiles, error } = await supabase
```

Comment:
Does Supabase still return an array if you do `limit(1)`?

Reply:
Yes, still an array.